features/admin.extensions.v1/configs/identity-provider.tsx

```diff
@@ -359,8 +359,14 @@ export const identityProviderConfig: IdentityProviderConfig = {
 
             return identityClaimsHiddenAuthenticators.has(authenticatorId);
         },
-        hideLogoInputFieldInIdPGeneralSettingsForm(): boolean {
-            return true;
+        hideLogoInputFieldInIdPGeneralSettingsForm(templateId: string): boolean {
```

Comment:
Let's move this to a util file. With the unification effort, we are trying to move away from extension configs.

Reply:
[2b93c04](https://github.com/wso2/identity-apps/pull/6440/commits/2b93c0440990c6b00f656f576af5b5e6f09c804c)